src/include/miopen/solver.hpp

```diff
@@ -2345,6 +2364,7 @@ struct GemmFwd1x1_0_2 : GemmFwdBase
     {
         return GetWorkspaceSize(ctx, ctx.conv_problem);
     }
+    bool MayNeedWorkspace() const { return true; }
```

Comment:
Wouldn't it be better to put MayNeedWorkspace() in the base class (GemmFwdBase)?

Reply:
That would create implicit dependence. `GemmFwdBase` does not provide a "base" implementation of  GetWorkspaceSize() and does not know if its successors need workspace.